functions/go/create-setters/README.md

```diff
@@ -0,0 +1,135 @@
+# create-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Add setter comments to matching resource fields. Setters serve as parameters for template-free setting of comments.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Setters are a safer alternative to other substitution techniques which do not
+have the context of the structured data. Setter comments can be added to 
+parameterize the field values of resources using create-setters function.
+
+We use ConfigMap to configure the `create-setters` function. The desired setter
+values are provided as key-value pairs using `data` field.
+Here, the key is the name of the setter which is used to set the comment and value is the field value to parameterize.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: create-setters-fn-config
+data:
+  setter_name1: setter_value1
+  setter_name2: setter_value2
+```
+
```

Comment:
**Note:** If this function adds setter comments to fields for which you din't intend to parameterize, you can simply review and delete those comments manually.